Massive update documentation for PowerSimulations (#1099)
NREL-Sienna/PowerSimulations.jl: docs/src/formulation_library/Branch.md

# `PowerSystems.Branch` Formulations

!!! note

>     The usage of reactive power variables and constraints will depend on the network model used, i.e. if it uses (or not) reactive power. If the network model is purely active power based, then no variables and constraints related to reactive power are created. For the sake of completion, if the formulation allows the usage of reactive power it will be included.

Reviewer: ```suggestion
    The use of reactive power variables and constraints will depend on the network model used, i.e., whether it uses (or does not use) reactive power. If the network model is purely active power-based,  reactive power variables and related constraints are not created.
```
